docs/en/observability/synthetics-private-location.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[synthetics-private-location]]
+= Add a private location
+
+IMPORTANT: This is only relevant to monitors created using the {uptime-app} or project monitors.
+
+beta[]
+
+Private locations allow you to run monitors from your own premises.
+	
+Before you create a private location, you need to set up {fleet}, and connect an agent to it.
+Then, you can create a private location in the {uptime-app} UI.
```

Comment:
NIT: Is it worth mentioning that the agent is who's going to run the monitors in these private locations or is that obvious enough?